loader_hub/asana/base.py

```diff
@@ -0,0 +1,63 @@
+"""Asana reader."""
+import logging
+from typing import List
+
+from gpt_index.readers.base import BaseReader
+from gpt_index.readers.schema.base import Document
+
+
+class AsanaReader(BaseReader):
+    """Asana reader. Reads data from an Asana workspace.
+
+    Args:
+        asana_token (str): Asana token.
+        asana_workspace (str): Asana workspace.
+    """
+
+    def __init__(self, asana_token: str) -> None:
+        """Initialize Asana reader."""
+        try:
```

Comment:
No need to do a try/except here anymore since `requirements.txt` is installed by `download_loader` prior to this. You can just `import asana` here in the `__init__` function.

Reply:
Done